genesyscloud/task_management_worktype/data_source_genesyscloud_task_management_unit_test.go

```diff
@@ -0,0 +1,73 @@
+package task_management_worktype
+
+import (
+	"context"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v121/platformclientv2"
+	"github.com/stretchr/testify/assert"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	"testing"
+)
+
+// TestAccDataSourceTaskManagementWorktypeStatus tests the retrieve of a status id.
```

Comment:
Decided to make this a unit test because I already have a test that covers almost all of this test.